airmozilla/base/templates/500.html

```diff
@@ -5,15 +5,55 @@
     <title>Something went wrong!</title>
     <style type="text/css">
       body {
-        font-family: "Open Sans Light","Lucida Sans","Lucida Grande","Lucida Sans Unicode",Verdana,sans-serif;
-        background-color: rgb(32, 39, 42);
-        color: rgb(255, 255, 255);
+        font-family: Verdana,sans-serif; 
+        background-color:#20272A;
+        background-image: url("https://air.cdn.mozilla.net/static/main/img/bg-midnight.png");
+        text-align: center;
+        position: relative;
+        padding: 0px 0px 84px;
+        background-position: center top;
+        background-repeat: repeat-x;
+        min-height: 550px;
       }
+      h1, h2, p {
+        color:#B4B4B4;
+        background: none repeat scroll 0% 0%;
+      }
+      h3 {
+        color:red;
+      }
+      #bugzilla {
+        position:absolute;
+        width:100%;
+        bottom:15%;
+        text-align: center;
+      }
+      #irc {
+        position:absolute;
+        width:100%;
+        bottom:5%;
+        text-align: center;
+      }
+      a {
+        color:#73BAE8;
+      } 
     </style>
   </head>
   <body>
-    <h1>Server Error</h1>
+    <h1>
+      <a href="https://air.mozilla.org/">
+        <img src="https://air.cdn.mozilla.net/static/img/logo-airmo-md.png">
+      </a>
+    </h1>
+    <h2>Server Error</h2>
+    <h3>OOPS!! &nbsp; Error 500</h3>
```

Comment:
This line doesn't help. The headline above said what was wrong which was "Server Error". The "oops" isn't helping. 
I would just delete this line.